resources/assets/components/StatusButton/status-button.scss

```diff
@@ -7,7 +7,11 @@ $accepted-green: #80E85D;
 
 .button {
   &.-accepted {
-    background: $light-gray;
+    background: $white;
+    color: $light-gray;
+    border-color: $light-gray;
+    border-width: 3px;
```

Comment:
The button border in only for the inactive state and should disappear when it gets accepted or rejected.